locales/en/messages.json

```diff
@@ -2865,8 +2865,17 @@
     "mixerOverrideEnableSwitchText": {
         "message": "Mixer Override forces the mixer inputs to arbitrary positions for setup purposes. The override values are not saved, and will be reset in reboot."
     },
+    "mixerPassthroughEnableSwitch": {
+        "message": "<strong>Enable Mixer Passthrough</strong>"
+    },
+    "mixerPassthroughEnableSwitchText": {
+        "message": "Mixer Passthrough allows you to control the mixer directly from TX. This mode will be reset in reboot."
+    },
```

Comment:
It appears this message does not get used. Maybe add it in a tooltip to the primary label.

Reply:
Oops.

I can't find a way to place it nicely so I removed this text and add it to the override help text.